nengo_gui/components/value.py

```diff
@@ -22,8 +23,13 @@ def __init__(self, obj):
         # the pending data to be sent to the client
         self.data = []
 
-        # the number of data values to send
-        self.n_lines = int(obj.size_out)
+        # grab the output of the object
+        self.output = self.obj
+        if isinstance(self.obj, spa.module.Module):
+            self.output = self.obj.outputs['default'][0]
```

Comment:
My earlier comment on this line seems to have disappeared; here it is again:

> This will crash if the module doesn't have a default output (there aren't any right now that don't but we may as well check for it now)